src/test/java/com/commercetools/sync/producttypes/ProductTypeSyncTest.java

```diff
@@ -447,6 +450,79 @@ void sync_WithInvalidAttributeDefinitions_ShouldThrowError() {
         assertThat(productTypeSyncStatistics).hasValues(1, 0, 0, 1, 0);
     }
 
+    @Test
+    void sync_WithErrorOnResolvingMissingReference_ShouldThrowError() {
```

Comment:
```suggestion
    void sync_WithErrorOnResolvingMissingReference_ShouldCallErrorCallback() {
```

Doesn't look like it throws error?

Reply:
Also the error message says: `Failed to update product type with key` and you're mocking throwing error on `updateProductType` -> doesn't look like `ErrorOnResolvingMissingReference` for me.

Reply:
78094524